### ISSUE TITLE
Support for updating multi-line strings with encoded yaml

### ISSUE THREAD

**Issue by inianv** [mikefarah/yq#974]
How much effort would be needed to add supporting modification of fields within YAML multi-line blocks. Is this supported in the underlying go YAML parser?

Supposing I have an YAML like below, where the `.values` is a YAML multi-line block. 

```yaml
---
apiVersion: argoproj.io/v1alpha1
kind: Application
metadata:
  namespace: mynamespace
  name: my-app
spec:
    helm:
      values: |-
        image:
          tag: "mytag"
          repository: "myrepo-image"
          registry: "myregistry"
```

The way, that we could right now is, filter out the block string as a YAML resource, apply filter to modify it and write it back to the original source. Something like

```bash
o="$(yq e '.spec.helm.values' yaml | yq e '.image.tag="footag"' -)" yq e '.spec.helm.values = strenv(o)' yaml
```

which as you can see involves 3 separate calls to `yq` and a bash command substitution involved. Can this be simplified?

**Comment by mikefarah** [comment]
Not the first time this is being asked for, I'm going to add `to_yaml/from_yaml` operators, as well as the json and prop equivalents in the next release. This will allow you to do the update in a single yq call.

**Comment by mikefarah** [comment]
Fixed in 4.14.1 - check out https://mikefarah.gitbook.io/yq/operators/encode-decode
